dammakaru/testing
# update (#40)

* test 123`

123

* Hello

thiis just test!!!!!

* test 34

අලියා

* test zxcbnj

hellloo 7896658

* deleted

check delete

* CLeanup

* Deleted Original DBconnect

* Delete MainClass.java

* Add files via upload

* Update DBConnect.java

* Added Main Panels for Admin and Manager + improved login

login -> exits on panel opening
one login for all panels

* commit.. (#22)

* Damsith (#24)

* new package

* deleted unwanted interfaces

* Damsith (#25)

* new package

* deleted unwanted interfaces

* generate in center

* new class to track user access level (#26)

* new package

* deleted unwanted interfaces

* generate in center

* made login the main class

* added new class to track access level

* Damsith (#27)

* new package

* deleted unwanted interfaces

* generate in center

* made login the main class

* added new class to track access level

* added "display current user"

* Delete MainInterface.java

* Delete MainInterface.form

* Add files via upload

* Delete InterfaceMain.java

* Delete InterfaceMain.form

* gbcollect

* gbcollect (#29)

* added warehouseManager Interface

* Added warehouse manager interface (#30)

* gbcollect

* added warehouseManager Interface

* Create tester

* Delete tester

* Merge remote-tracking branch 'refs/remotes/origin/master' into dilshanka (#31)

* @naveenluke (#32)

* test 123`

123

* Hello

thiis just test!!!!!

* test 34

අලියා

* test zxcbnj

hellloo 7896658

* deleted

check delete

* Damsith (#34)

* gbcollect

* added warehouseManager Interface

* deleted unwanted class (#35)

* Dilshanka (#38)
